dolphinscheduler-api-test/dolphinscheduler-api-test-case/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to Apache Software Foundation (ASF) under one or more contributor
+  ~ license agreements. See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright
+  ~ ownership. Apache Software Foundation (ASF) licenses this file to you under
+  ~ the Apache License, Version 2.0 (the "License"); you may
+  ~ not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dolphinscheduler-api-test</artifactId>
+        <groupId>org.apache.dolphinscheduler</groupId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
I think we should keep the version consistent

Reply:
Should we use ```dev-SNAPSHOT``` in this? @kezhenxu94 @caishunfeng @zhongjiajie

Reply:
> Should we use `dev-SNAPSHOT` in this? @kezhenxu94 @caishunfeng @zhongjiajie

I don't think it necessary. `-api-test` and `-e2e` are out of the project hierarchy of the main project so we don't bother to make it consistent